tests-manual/debugger/step

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#help 		Allow readings to flow into the pipeline. Passed an optonal number of readings to ingest, default to 1 if omitted
```

Comment:
Passed an optonal number of readings to ingest, default to 1 if omitted -> Passed an optional number of readings to ingest; default to 1 if omitted